README.md

```diff
@@ -352,33 +371,45 @@ This endpoint will send your order to the restaurant and cannot be taken back.
                     		//do stuff with your data
                 	 })
        		 })
-
+```
 
 ##### Success Response
+```json
 	{
-		"success": true,
-		"message": "Order submitted",
-		"status": 200,
-		"data": [
-					{
-						"_id": "60c73afb0b5f5c23d4a61688",
-						"name": "Ashley Coles",
-						"deliveryAddress": "BA2 6AH",
-						"email": "deliciousFood@food.com",
-						"isOrderSubmitted": true,
-						"timePlaced": "2000-01-01T00:00:00.000+00:00"
-						"orderItems": [{},{}]
-                    }
-				],
-		"deliveryTime": "2000-01-01T00:00:00.000+00:00"
-	}
+        "success": true,
+        "message": "The order has been placed",
+        "status": 200,
+        "data": {
+            "_id": "60c73afb0b5f5c23d4a61688",
+            "name": "isabelle",
+            "deliveryAddress": "bse 93b",
+            "email": "isabelle@hotmail.co.uk",
+            "orderItems": [],
+            "isOrderSubmitted": true,
+            "timePlaced": "2021-06-17T08:42:53.925Z",
+            "totalPrice": 0
+          }
+    }
+```
 
-*this success response may change 
+##### Error Responses
 
-##### Error Response
+If the order has already been submitted
+
+```json
+	{
+		"success": false,
+		"message": "Order has already been submitted",
+		"status": 404
+	}
+```
 
+If there is an issue connecting to the database
+
+```json
 	{
 		"success": false,
-		"message": "The resource/s requested does not exist at the desired location.",
+		"message": "Database request failed",
 		"status": 404
 	}
```

Comment:
please remove some of the indentation so that the curly braces are against the left side of the page